examples/plot_kmean_poincare_disk.py

```diff
@@ -0,0 +1,75 @@
+"""
+Plot a square on H2 with Poincare Disk visualization
+with two clusters from uniformly sampled random points.
+Then apply the K-means algorithm and plot the points labels
+as two distinct colors.
+"""
+import os
+
+import matplotlib.pyplot as plt
+
+import geomstats.backend as gs
+import geomstats.visualization as visualization
+from geomstats.geometry.hyperbolic_space import HyperbolicMetric
+from geomstats.geometry.hyperbolic_space import HyperbolicSpace
+from geomstats.learning.k_means import RiemannianKMeans
+
+SQUARE_SIZE = 50
+
+
+def main():
+
+    Cluster_1 = gs.random.uniform(low=0.5, high=0.6, size=(20, 2))
```

Comment:
no uppercase letters in vairable names (only classes have uppercases) as per pep8 coding style